packages/experimental/kai/vite.config.ts

```diff
@@ -104,30 +104,30 @@ export default defineConfig({
 
     // TODO(burdon): Document.
     // To reset, unregister service worker using devtools.
-    VitePWA({
-      workbox: {
-        maximumFileSizeToCacheInBytes: 30000000
-      },
-      includeAssets: ['favicon.ico'],
-      manifest: {
-        name: 'DXOS Kai',
-        short_name: 'Kai',
-        description: 'DXOS Kai Demo',
-        theme_color: '#ffffff',
-        icons: [
-          {
-            src: 'icons/icon-32.png',
-            sizes: '32x32',
-            type: 'image/png'
-          },
-          {
-            src: 'icons/icon-256.png',
-            sizes: '256x256',
-            type: 'image/png'
-          }
-        ]
-      }
-    }),
+    // VitePWA({
```

Comment:
i might just have a static conditional here -- e.g., boolean var at the top. just to keep the code "active" rather than commented.